.storybook/preview.tsx

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
-import type { Preview } from '@storybook/react';
 import { MockedProvider } from '@apollo/client/testing';
 
 import '~utils/yup/customMethods';
 import '../src/styles/main.global.css';
 import { reactIntl } from './reactIntl.js';
 import { applyTheme } from "../src/components/frame/Extensions/themes/utils";
 
-const preview: Preview = {
```

Comment:
Why tho? This is actually helpful as it verifies if all options have the correct types/shape

Reply:
You're right, added back